index.html

```diff
@@ -44,237 +44,275 @@
-						<h2>Are the developers who wrote a piece of code responsible for its operation in production?</h2>
-					</section>
-
-					<section>
-						<h2>Do you enforce age limits for all deployed systems and infrastructure components?</h2>
-					</section>
-
-					<section>
-						<h2>Do you have pipelines that safely release changes to production without human verification?</h2>
-					</section>
-				</section>
+        <section>
+          <section>
+            <h1>DOJO Scoring</h1>
+          </section>
+
+          <section>
+            <h3>Scale</h3>
+            <table>
+              <tr><td>0</td><td>Not planned</td></tr>
+              <tr><td>1</td><td>Planned</td></tr>
+              <tr><td>2</td><td>Inconsistently implemented in some areas</td></tr>
+              <tr><td>3</td><td>Consistently implemented in some areas</td></tr>
+              <tr><td>4</td><td>Consistently implemented throughout the organization</td></tr>
+            </table>
+          </section>
+
+          <section>
+            <h3>Two Passes</h3>
+            <ol>
+              <li>Current state</li>
+              <li>Target state</li>
+            </ol>
+          </section>
+
+          <section>
+            <h3>Score Each Statement</h3>
+            <h4>Example</h4>
+            <ol>
+              <li>Desired configuration is expressed in chef cookbooks</li>
+              <li>All configuration changes are made via chef-client runs</li>
+              <li>All chef cookbooks are in source control</li>
+            </ol>
+          </section>
+
+          <section data-transition="fade">
+            <h3>Score Each Statement</h3>
+            <h4>Example with Current State</h4>
+            <ol>
+              <li>Desired configuration is expressed in chef cookbooks 2</li>
+              <li>All configuration changes are made via chef-client runs 1</li>
+              <li>All chef cookbooks are in source control 1</li>
+            </ol>
+          </section>
+
+          <section data-transition="fade">
+            <h3>Score Each Statement</h3>
+            <h4>Example with Current State and Target State</h4>
+            <ol>
+              <li>Desired configuration is expressed in chef cookbooks 2 3</li>
+              <li>All configuration changes are made via chef-client runs 1 3</li>
+              <li>All chef cookbooks are in source control 1 3</li>
+            </ol>
+          </section>
+        </section>
+
+        <section>
+          <section>
+            <h1>Working with People</h1>
+            <ol>
+              <li>Organizational Success Factors</li>
+              <li>Organizational Culture</li>
+              <li>Data-Driven Decision Making</li>
+              <li>Coding Practices</li>
+            </ol>
+          </section>
+
+					<section>
+						<h3>Organizational Success Factors</h3>
+            <ol>
+              <li>Your IT organization widely recognizes IT as a competitive advantage to your business</li>
```

Comment:
wait, I might be confused because there's two questions now.   it's still awkward having IT twice,  maybe change one of them to technology?